tgui-next/packages/tgui/interfaces/Mint.js

```diff
@@ -13,10 +13,13 @@ export const Mint = props => {
         title="Materials"
         buttons={(data.processing ? (
           <Button
+            icon="times"
             content="Stop"
+            selected
```

Comment:
?

Reply:
![image](https://user-images.githubusercontent.com/33292112/69231944-52288100-0b8a-11ea-930b-4c065ad1249a.png)

Reply:
it does seem like a legitimate usage, although it would be less confusing if you changed props based on the condition instead of choosing between two buttons.

Reply:
Allright, I changed the buttons around